tests/Feature/ScoutElasticSearchServiceProviderTest.php

```diff
@@ -7,17 +7,13 @@
 
 class ScoutElasticSearchServiceProviderTest extends TestCase
 {
-    public function test_config_publishing()
+    public function test_config_is_merged_from_the_package()
     {
-        \File::delete(config_path('elasticsearch.php'));
```

Comment:
I think I know better way to test it. 
Revert your changes in this test. We don't need new test method
Delete this file `https://github.com/matchish/laravel-scout-elasticsearch/blob/master/tests/laravel/config/elasticsearch.php`
After it a lot of tests should fail
Add your code `$this->mergeConfigFrom(__DIR__.'/../config/elasticsearch.php` it should fix them because of now we load config from vendor package
And move this code to the end of the test for cleanup
`File::delete(config_path('elasticsearch.php'))`